tap_db2/__init__.py

```diff
@@ -1,10 +1,12 @@
 # import datetime
+from __future__ import generators
```

Comment:
I don't believe you need this lines. It is only required for Python 2.x which we don't support anyway. I didn't use this line in my implementation. Might we worth double checking my understanding of this though.